README.md

```diff
@@ -55,12 +55,13 @@ Note¹ - Please set 'Outdated Server' message to 'false' in the CustomAlert plug
 Note² - Please set 'Full server' message to 'false' in the CustomAlert plugin or any related plugin if you encounter *Full server*
 
 ## For Developers
-This is a clean, high quality code base. Developing/modifying this project is easy.<br>
+This is a clean, high quality code base, because we are trying to keep PocketMine's great code quality.
+Developing/modifying this project is easy.<br>
 We are still rewriting the base to make sure ClearSky has the best developing feel.<br>
 ClearSky is not just for a CLEAN feel for users - it's also for developers!<br>
 
 # <a name="SPA"></a>Español
-# Te presentamos: ClearSky-Sunrise (PHP5)
+# Te presentamos: ClearSky-Sunrise ~ A [PM](https://github.com/PocketMine/PocketMine-MP) fork (PHP5) 
```

Comment:
@robske110 You are a great person because you don't steal code from other person